doc/configuration.rst

```diff
@@ -1785,6 +1786,53 @@ flag is passed to ``sphinx-build``. This can be enabled by setting::
     }
 
 
+.. _recommend_examples:
+
+Enabling the example recommender system
+=======================================
+
+Sphinx-Gallery can be configured to generate content-based recommendations for
+the example gallery. A list of related examples is automatically generated by
+computing the closest examples in the `TF-IDF
+<https://en.wikipedia.org/wiki/Tf%E2%80%93idf>`_ space of their text contents.
+Then, the most similar content is displayed at the bottom of each example as a
+set of thumbnails.
+
+The recommender system can be enabled by setting ``enable`` to ``True``. To
+configure it, pass a dictionary to the ``sphinx_gallery_conf``, e.g.::
+
+    sphinx_gallery_conf = {
+        ...
+        "recommender": {"enable": True, "n_examples": 5, "min_df": 3, "max_df": 0.9},
+    }
+
+The only necessary parameter is ``enable``. If any other parameters is not
+specified, the default value is used. Below is a more complete explanation of
+each field:
+
+enable (type: bool, default: False)
+  Whether to generate recommendations inside the example gallery. Enabling this
+  feature requires adding `numpy` to the dependencies.
+n_examples (type: int, default: 5)
+  Number of most relevant examples to display.
+min_df (type: float in range [0.0, 1.0] | int, default: 3)
+  When building the vocabulary ignore terms that have a document frequency
+  strictly lower than the given threshold. If float, the parameter represents a
+  proportion of documents, integer represents absolute counts. This value is
+  also called cut-off in the literature.
+max_df (type: float in range [0.0, 1.0] | int, default: 0.9)
+  When building the vocabulary ignore terms that have a document frequency
+  strictly higher than the given threshold. If float, the parameter represents a
+  proportion of documents, integer represents absolute counts.
+rubric_header (type: str, default: "Related examples")
+  Customizable rubric footer, e.g. to link to API doc of the recommender system
```

Comment:
Is this meant to be 'header'? Could you expand on the 'e.g.'? Does it mean you can customise the header to add a link to API doc?

Reply:
Hopefully the wording introduced in [063df21](https://github.com/sphinx-gallery/sphinx-gallery/pull/1125/commits/063df2199432d2198560d4dd2567b25c3ee9fd75) is clearer.